pom.xml

```diff
@@ -54,7 +54,7 @@
         <package-final-name>${project.artifactId}-${project.version}</package-final-name>
 
         <!-- other packages that the documentation (of apps) links to; their `abbrev` must match their expath-pkg.xml -->
-        <documentation.pkg.abbrev>exist-documentation</documentation.pkg.abbrev>
+        <documentation.pkg.abbrev>doc</documentation.pkg.abbrev>
```

Comment:
I don't actually see this as a good thing. Let's discuss some more...

Reply:
let's not. If this one line fix doesn't solve the problem, we ll need to fix the build script and public-repo code properly. We still have broken links, and code despite having a separate name-my-app-app. Good indication that we're over-engineering hacks, without fixing the underlying problem.